packages/dialtone-vue2/package.json

```diff
@@ -92,7 +92,7 @@
     "typedoc-plugin-markdown": "^3.16.0",
     "vite": "^5.0.0",
     "vite-plugin-vue2-svg": "^0.4.0",
-    "vitest": "^0.34.5",
+    "vitest": "^1.0.4",
```

Comment:
You can remove the dependency here if it's already on root's package.json

Reply:
I think it's not inheriting, as you're running `vitest` from the root package, the other packages don't really need to have that dependency.